Task8

MyxaWanyuha opened the pull request.

MyxaWanyuha committed:
init

MyxaWanyuha committed:
task8

solodyankina commented on task4.sln:
```diff
@@ -9,6 +9,8 @@ Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "task6", "task6\task6.vcxpro
 EndProject
```
удалить лишние файлы

solodyankina commented on task8/Maze.cpp:
```diff
@@ -0,0 +1,54 @@
+    return i1 < i2 && cell(i1, j1).right() || j1 < j2 && cell(i1, j1).down();
+}
+
+bool Maze::makeConnection(int i1, int j1, int i2, int j2)
```
здесь и далее.. порядок передаваемых  значений не должен играть роли

solodyankina commented on task8/Maze.cpp:
```diff
@@ -0,0 +1,54 @@
+    {
+        for (int j = 0; j < m; j++)
+        {
+            if (hasConnection(i, j, i + 1, j))
```
только 2 значения?

MyxaWanyuha committed:
task8 fixes1

Добавил поддержку всех символов.
Добавил проверку, что клетки соединяемые/разъединяемые находятся рядом и не выходят за границу лабиринта.

MyxaWanyuha committed:
task8 fixes1.1

Удалил лишние файлы

MyxaWanyuha committed:
task8 fixes1.2 and task9

task8: удалил ненужную функцию в task8.cpp и исправил один метод
task9: выполнил

MyxaWanyuha committed:
delete task9

MyxaWanyuha committed:
task8 fixes1.3

Удалил все лишние функции, полное соответствие с тз

MyxaWanyuha committed:
add MCell.cpp

MyxaWanyuha committed:
fix delete

MyxaWanyuha committed:
task8 fixes 4

MyxaWanyuha committed:
task9

MyxaWanyuha committed:
task9 fix1

Toxin65 commented:
Тут тоже непонятно. Мне что-то надо смотреть, или уже неактуально?

MyxaWanyuha commented:
Уже не актуально
  
>Вторник, 22 декабря 2020, 13:32 +03:00 от Toxin65 <notifications@github.com>:
> 
> 
>Тут тоже непонятно. Мне что-то надо смотреть, или уже неактуально?
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
--
Иван Саблин
 

Toxin65 commented:
Что-то ещё надо смотреть, или всё уже отсмотрено?

MyxaWanyuha commented:
Все отсмотрено

  
>Вторник, 22 декабря 2020, 20:26 +03:00 от Toxin65 <notifications@github.com>:
> 
> 
>Что-то ещё надо смотреть, или всё уже отсмотрено?
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  unsubscribe . 
 
 
--
Иван Саблин
 

Toxin65 commented:
Принято.
То есть 10ая работа у вас отсутствует, согласно нашей документации.
Прошу подтвердить, если так, либо опровергнуть.
Спасибо!
